warehouse/redshift/redshift.go

```diff
@@ -21,35 +21,43 @@ import (
 	warehouseutils "github.com/rudderlabs/rudder-server/warehouse/utils"
 )
 
-var (
+type Config struct {
 	setVarCharMax                 bool
 	dedupWindow                   bool
 	dedupWindowInHours            time.Duration
-	pkgLogger                     logger.Logger
 	skipComputingUserLatestTraits bool
 	enableDeleteByJobs            bool
-)
+}
```

Comment:
Instead of defining a new config struct, why not use the existing `Handler` struct to store those things?

Reply:
Also, we would probably need to expose those fields so we can set them on our test suite

Reply:
I thought injecting a mock Config would be easy while testing.

Reply:
But we can add this in Handler as well.